Binance Detect Moonings.py

```diff
@@ -383,14 +391,20 @@ def sell_coins():
 
             # try to create a real order
             try:
+                try:
+                    rounded_amount = round_step_size(coins_bought[coin]['volume'], coins_bought[coin]['step_size'])
+                except Exception:
+                    tick_size = float(next(
+                        filter(lambda f: f['filterType'] == 'LOT_SIZE', client.get_symbol_info(coin)['filters'])
+                    )['stepSize'])
```

Comment:
This line seems to be too long. Is there any way to make it more readable? (I really don't know)

Reply:
Haven't tested this yet, will it leave crypto dust for people using BNB for fees?

Reply:
Yep, it will leave crypto dust which can be changed into BNB afterward.